resources/assets/less/bem/user-group-badge.less

```diff
@@ -5,7 +5,7 @@
   @_top: user-group-badge;
   border-radius: 20px;
   font-size: 10px;
-  background-color: osu-hsla(@osu-colour-b6, 0.75);
+  background-color: osu-hsla(@osu-colour-b6, 0.9);
```

Comment:
I don't think the opacity of the background needs to be adjusted if the `probationary` modifier is adjusting the whole opacity?